### ISSUE TITLE
OPNsense installation rDNS / ARP for client mapping not working

### ISSUE THREAD

**Issue by werkkrew** [AdguardTeam/AdGuardHome#5070]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

OpenBSD

### CPU architecture

AMD64

### Installation

Custom port

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

v0.107.16

#### What did you do?
- Configured opnsense DHCP with several static leases and associated hostnames
- Configured unbound to register DHCP static lease names 
- Verified using dig and nslookup that these names do indeed exist, e.g. 192.168.1.50 might resolve to test_host
- Verified this hostname is also visible in the opnsense ARP table
- Configured AGH to use unbound (127.0.0.1:5353) as reverse lookup DNS server for private reverse name resolution

#### Expected result
- AGH reports IP's such as 192.168.1.50 as test_host in the logs and client table without the need for a manual persistent client definition

#### Actual result
Runtime clients list is essentially blank, I have to enter all clients by hand in the persistent clients fields to get useful logs and reporting info.

#### Screenshots (if applicable)
https://i.imgur.com/x9N2yQX.png

https://i.imgur.com/HUpGiEX.png

https://i.imgur.com/6SmAmvy.png

**Comment by agneevX** [comment]
Just to be clear, something like `dig +short -x 192.168.1.184 @localhost -p 5353` returns a correct result?

**Comment by werkkrew** [comment]
Yes, my opnsense box doesnt have dig on it so I ran it from another box substituting @localhost for the unbound ip:

https://i.imgur.com/pPWHp4L.png

**Comment by agneevX** [comment]
Is there anything in query log?

**Comment by werkkrew** [comment]
Sure, the query log appears to work fine, but I have mapped most of the
clients persistently.  Nothing related to the reverse dns shows up in the
query log.
